src/extension/wmts-multi-dimensional/src/test/java/org/geoserver/gwc/wmts/RasterCustomDimensionTest.java

```diff
@@ -58,6 +58,35 @@ public void testDisabledDimension() throws Exception {
                 DimensionsUtils.extractDimensions(wms, getLayerInfo(), ALL_DOMAINS).size(), is(0));
     }
 
+    @Test
+    public void testRasterCustomFixedValueDimension() throws Exception {
```

Comment:
Do we really need this test for the fixed values string? Seems a simply copy paste from another test with just the addition of the fixed value string to the DimensionInfo object. Not clear what is the functional meaning of this.